src/tools/PrizeTierController/PrizeTierHistoryList.tsx

```diff
@@ -0,0 +1,291 @@
+
+  // Calculating combined outcome of existing and edited prize tiers:
+  const combinedPrizeTier = formatCombinedPrizeTier(prizeTier, prizeTierEdits)
+  useEffect(() => {
+    setCombinedPrizeTiers(() => {
+      const updatedCombinedPrizeTiers = { ...combinedPrizeTiers }
+      if (!updatedCombinedPrizeTiers[prizePool.chainId]) {
+        updatedCombinedPrizeTiers[prizePool.chainId] = {}
+      }
+      updatedCombinedPrizeTiers[prizePool.chainId][prizePool.prizeTierHistoryMetadata.address] =
+        combinedPrizeTier
+      return updatedCombinedPrizeTiers
+    })
+  }, [prizeTier, prizeTierEdits])
+
+  const defaultNumberofPrizesPerTier = calculate.calculateNumberOfPrizesPerTier(prizeTier)
+  const defaultTotalPrizes = defaultNumberofPrizesPerTier.reduce((a, b) => a + b, 0)
+
+  const numberOfPrizesPerTier = calculate.calculateNumberOfPrizesPerTier(combinedPrizeTier)
+  const valueOfPrizesPerTier = combinedPrizeTier.tiers.map((tier, index) =>
+    calculate.calculatePrizeForTierPercentage(
+      index,
+      tier,
+      combinedPrizeTier.bitRangeSize,
+      combinedPrizeTier.prize
+    )
+  )
+  const totalPrizes = numberOfPrizesPerTier.reduce((a, b) => a + b, 0)
+
+  const [seeMore, setSeeMore] = useState(false)
+  const setIsOpen = useUpdateAtom(isEditPrizeTiersModalOpenAtom)
+  const setSelectedPrizePoolId = useUpdateAtom(selectedPrizePoolIdAtom)
+  const setSelectedPrizeTierHistoryAddress = useUpdateAtom(selectedPrizeTierHistoryAddressAtom)
+  const setSelectedPrizeTierHistoryChainId = useUpdateAtom(selectedPrizeTierHistoryChainIdAtom)
+
+  const [isCollapsed] = useAtom(isPrizeTierListCollapsed)
+  useEffect(() => {
+    setSeeMore(!isCollapsed)
+  }, [isCollapsed])
+
+  return (
+    <div>
+      <div className={classNames('grid grid-cols-2 gap-x-2 gap-y-3', { 'mb-4': seeMore })}>
+        <Stat label='Total Prizes' value={totalPrizes} defaultValue={defaultTotalPrizes} />
+        <Stat
+          label='Total Value'
+          value={formatUnformattedBigNumberForDisplay(
+            combinedPrizeTier.prize,
+            tokens?.token.decimals
+          )}
+          defaultValue={formatUnformattedBigNumberForDisplay(
+            prizeTier.prize,
+            tokens?.token.decimals
+          )}
+        />
+        {seeMore && (
+          <PrizeTierStats
+            prizeTier={combinedPrizeTier}
+            defaultMaxPicksValue={prizeTier.maxPicksPerUser}
+            defaultBitRangeValue={prizeTier.bitRangeSize}
+          />
+        )}
+      </div>
+
+      {seeMore && (
+        <PrizesList
+          prizePool={prizePool}
+          prizeTier={combinedPrizeTier}
+          numberOfPrizesPerTier={numberOfPrizesPerTier}
+          valueOfPrizesPerTier={valueOfPrizesPerTier}
+        />
+      )}
+
+      <div className={classNames('w-full flex justify-between', { 'mt-4': seeMore })}>
+        <button onClick={() => setSeeMore(!seeMore)}>See {seeMore ? 'less' : 'more'}</button>
```

Comment:
This can go away now that there's the expand/collapse all buttons

Reply:
Removed in 38d0db1